src/module_PI/Raspberry_PI/main/DataPacket.java

```diff
@@ -60,6 +65,12 @@ private void parseLogLevelFromByteArray(){
 		packetLogLevel = PacketLogLevel.getLogLevelFromByte(packetCodeByteArray[LOG_LEVEL_INDEX]);
 	}
 	
+	private void parseMessage(){
+		int packetIntModThisIsMessageInt = (int) Math.pow(10, NUM_OF_MESSAGE_BYTES_IN_PACKET);
```

Comment:
Lol I spent a good 5 minutes thinking about what to call that stupid thing. I could just hard code packetInt % 1000, but I see that as being a source of frustration if we ever decide we need an extra digit or something.